superset/dashboards/commands/export.py

```diff
@@ -37,6 +40,53 @@
 JSON_KEYS = {"position_json": "position", "json_metadata": "metadata"}
 
 
+def suffix(length: int = 8) -> str:
+    return "".join(
+        random.SystemRandom().choice(string.ascii_uppercase + string.digits)
+        for _ in range(length)
+    )
+
+
+def default_position(title: str, charts: List[Slice]) -> Dict[str, Any]:
+    chart_hashes = [f"CHART-{suffix()}" for _ in charts]
+    row_hash = f"ROW-N-{suffix()}"
+    position = {
+        "DASHBOARD_VERSION_KEY": "v2",
+        "ROOT_ID": {"children": ["GRID_ID"], "id": "ROOT_ID", "type": "ROOT",},
+        "GRID_ID": {
+            "children": [row_hash],
+            "id": "GRID_ID",
+            "parents": ["ROOT_ID"],
+            "type": "GRID",
+        },
+        "HEADER_ID": {"id": "HEADER_ID", "meta": {"text": title,}, "type": "HEADER",},
+        row_hash: {
+            "children": chart_hashes,
+            "id": row_hash,
+            "meta": {"0": "ROOT_ID", "background": "BACKGROUND_TRANSPARENT",},
+            "parents": ["ROOT_ID", "GRID_ID"],
+            "type": "ROW",
+        },
+    }
+
+    for chart_hash, chart in zip(chart_hashes, charts):
+        position[chart_hash] = {
+            "children": [],
+            "id": chart_hash,
+            "meta": {
+                "chartId": chart.id,
+                "height": 50,
```

Comment:
i'd make height and width constants at the top

Reply:
Good point, let me do that!